mycroft/skills/__main__.py

```diff
@@ -24,9 +24,14 @@
 from mycroft import dialog
 from mycroft.api import is_paired, BackendDown, DeviceApi
 from mycroft.audio import wait_while_speaking
-from mycroft.enclosure.api import EnclosureAPI
 from mycroft.configuration import Configuration, setup_locale
+from mycroft.enclosure.api import EnclosureAPI
 from mycroft.messagebus.message import Message
```

Comment:
should this import from `mycroft_bus_client` directly? `mycroft.messagebus` should just be wrapping those imports anyways

Reply:
no, core expects that Message object contains `utterance_remainder`, we must import the patched version. 

api compatibility once more....